subsys/net/ip/Kconfig.ipv4

```diff
@@ -12,10 +12,27 @@ menuconfig NET_IPV4
 if NET_IPV4
 
 config NET_INITIAL_TTL
-	int "Initial time to live for a connection"
+	int "Initial IPv4 time to live value for a connection"
```

Comment:
`for a connection` sounds a bit awkward in IP context, maybe `for unicast packets` similar as below, so it's explicit that the config is now only about unicast?